praise-bot/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the Ubtuntu base image
+FROM ubuntu:22.04
+
+# Download development tools
+RUN apt-get update
+RUN apt-get install -y curl
+RUN apt-get install -y git
+RUN apt-get install -y unzip
+
+# Run the container as a user
+RUN useradd -m -s /bin/bash slackdev
+RUN chown slackdev /usr/local/bin
+USER slackdev
+
+# Install the project runtime
+RUN curl -fsSL https://deno.land/install.sh | sh
+RUN export DENO_INSTALL="/home/slackdev/.deno"
```

Comment:
What's this for @AndrewHUNGNguyen ? You normally don't want hardcoded directories in Dockerfiles

Reply:
Hi Greg, I think this is an auto-generated file when I copied the starter code.